src/openzaak/components/zaken/admin/zaken.py

```diff
@@ -684,6 +687,32 @@ class ZaakAdmin(
     viewset = "openzaak.components.zaken.api.viewsets.ZaakViewSet"
     widget = AuthorityAxisOrderOLWidget
 
+    @admin.display(description="Zaaktype")
+    def get_zaaktype(self, obj) -> str:
+        if not obj._zaaktype:
+            return ""
+
+        return obj._zaaktype.identificatie
+
+    @admin.display(description="Resultaat")
+    def get_resultaat(self, obj) -> str:
+        try:
+            resultaat = Resultaat.objects.get(zaak=obj, _resultaattype__isnull=False,)
+            resultaattype = resultaat._resultaattype
+            return resultaattype.omschrijving
+        except Resultaat.DoesNotExist:
+            return ""
+
+    @admin.display(description="Status")
+    def get_status(self, obj) -> str:
+        status = obj.current_status
+
+        if not status or not status._statustype:
+            return ""
+
+        statustype = status._statustype
+        return statustype.statustype_omschrijving
+
```

Comment:
Could you optimize the queryset a bit? It's quite slow now with a lot of duplicate queries
![image](https://github.com/open-zaak/open-zaak/assets/29249171/726fd4f0-3fbb-4777-acc3-afb29a698e83)